tutorials/tour/_posts/2017-02-13-mixin-class-composition.md

```diff
@@ -9,28 +9,41 @@ categories: tour
 num: 6
 next-page: anonymous-function-syntax
 previous-page: traits
+prerequisite-knowledge: inheritance, traits, abstract-classes, unified-types
 ---
+Mixin class composition means that a class can reuse the members of one or more mixins (i.e. traits and types).
+
+```tut
```

Comment:
Another idea is to change 'def' to 'val'. I don't have a strong preference here but it simplifies things a bit.